lib/trln_argon/view_helpers/items_section_helper.rb

```diff
@@ -109,6 +109,14 @@ def suppress_item?(options = {})
           loc_b.blank? ||
           item_data.fetch('items', []).empty?
       end
+
+      def get_item_id(item)
+        item.key?('item_id') ? item['item_id'] : ''
+      end
+
+      def assign_item_id_as_id(item)
+        item.key?('item_id') ? ('id="item-' + item['item_id'] + '"').html_safe : ''
```

Comment:
Prefer interpolation over concatenation:
`item.key?('item_id') ? ("id=\"item-#{item['item_id']}\"").html_safe : ''`

I wonder if we will run into issues with characters in the item_id that require HTML escaping? I think there are odd characters in the IDs from time to time.

Reply:
Thank you @corylown ! Code has been updated accordingly.